### PR TITLE
Signup edits

### PR DESCRIPTION
**Pull request by billc2501**

### CODE REVIEW

**Review by areed08 — APPROVED** [review]
Looks good! A couple of your urls are "http" instead of "https" but I don't think that's a big deal. And it passes Travis so yay!

**Review by dathash — APPROVED** [review]
Looks good, no issues here.

File: src/components/Signup.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      });
+      setToken(token);
+      });}
+    
+
+
+
+  return(
+    <div className="signup-wrapper">
+      <a href="http://localhost:3000/">
```
Comment (KennaReed) [inline src/components/Signup.js]: Is this supposed to be a heroku http rather than a localhost?

File: src/components/Signup.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      for (i=0; i < (json.account_list.length); i ++){
+        console.log(json.account_list[i]);
+        console.log(username);
+        if (json.account_list[i].username === username){
+            if (json.account_list[i].password !== password){
+              check = true;
+            }
+        }
+      }
+      if (check === true){
```
Comment (KennaReed) [inline src/components/Signup.js:38]: What if check is not true? Ie password does not match password for given user? Is there any sort of error handling?

**Review by KennaReed — APPROVED** [review]
Looks good! Left a couple questions in the sign up doc that we can talk about in our meeting tom.

**Review by jackson14furie — APPROVED** [review]
Looks good! Go ahead and push it.

File: src/components/Signup.js (hunk 1 above)
Comment (billc2501) [reply src/components/Signup.js]: has been updated

File: src/components/Signup.js (hunk 2 above)
Comment (billc2501) [reply src/components/Signup.js:38]: it should move on the other condition as intended

**Pull request merged**